src/GovUk.Education.ExploreEducationStatistics.Common/Constants/ValidationConstants.cs

```diff
@@ -0,0 +1,25 @@
+namespace GovUk.Education.ExploreEducationStatistics.Common.Constants;
+
+public class ValidationConstants
+{
+    public const int TitleMaxLength = 120;
+    public const string TitleMaxLengthMessage = "Title must be 120 characters or less";
+
+    public const int SummaryMaxLength = 250;
+    public const string SummaryMaxLengthMessage = "Summary must be 250 characters or less";
+
+    public const int SubjectTitleMaxLength = 120;
+    public const string SubjectTitleMaxLengthMessage = "Subject title must be 120 characters or less";
+
+    public const int TableTitleMaxLength = 120;
+    public const string TableTitleMaxLengthMessage = "Table title must be 120 characters or less";
+
+    public const int FeaturedTableNameMaxLength = 120;
+    public const string FeaturedTableNameMaxLengthMessage = "Featured table name must be 120 characters or less";
+
+    public const int FeaturedTableDescriptionMaxLength = 200;
+    public const string FeaturedTableDescriptionMaxLengthMessage = "Featured table description must be 200 characters or less";
+
+    public const int FileGuidanceContentMaxLength = 250;
+    public const string FileGuidanceContentMaxLengthMessage = "File guidance content must be 250 characters or less";
+}
```

Comment:
I think this could get pretty unwieldy really quickly if we continue with this pattern.

For example, `TitleMaxLength` and `SummaryMaxLength` - it's not obvious which title and summary fields these relate to.

There will become a namespacing issue as more and more features want to add their validation constants to this file where we have to come up with a strategy for grouping related constants together. Do we create sub classes, or create groups under code comments, etc?

Having this class located in `Common` can also expose constants to more projects than is necessary.

I think it would be better to leave the constants inline with the FluentValidation validator config, where its then obvious what the character limits are, without having to reach out to this file, and we avoid this file growing to a point where it gets unmanageable.

Reply:
Inline was my preference, but the `ReleasesController` methods use query parameters rather than a model. This meant that I couldn't replace the validation with Fluent validation to match the rest of the code, and the values used in the data annotations couldn't be anything other than constants.

Reply:
Removed as per other comment.